components/user_settings/display/user_settings_theme/custom_theme_chooser.jsx

```diff
@@ -129,6 +129,11 @@ export default class CustomThemeChooser extends React.PureComponent {
         this.state = {
             copyTheme,
         };
+
+        this.textarea = React.createRef();
```

Comment:
In order to have consistency with other PRs, can we rename the reference? Can you do the same with the other refs?
```suggestion
        this.textareaRef = React.createRef();
```